features/action_chain.feature

```diff
@@ -58,20 +58,6 @@ Feature: Test action chaining
     And I click on "Confirm"
     Then I should see a "Action has been successfully added to the Action Chain" text
 
-  Scenario: I add a config file deployment to the action chain
```

Comment:
Why not create a cleanup scenario to remove the deployed files instead of removing the entire scenario?